tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,84 @@
+### Reasonable solution
+
+- using Dictionary
+    ```csharp
+    using System.Collections.Generic;
+
+    public class SpaceAge
+    {
+        private readonly double earthYear;
+
+        private static Dictionary<string, double> conversionFactor 
+                = new Dictionary<string, double>()
+        {
+            {"Mercury", 0.2408467},
+            {"Venus",   0.61519726},
+            {"Mars",    1.8808158},
+            {"Jupiter", 11.862615},
+            {"Saturn",  29.447498},
+            {"Uranus",  84.016846},
+            {"Neptune", 164.79132},
+        };
+
+        public SpaceAge(int seconds)
+        {
+            earthYear = seconds / 31557600.0;
```

Comment:
I like how the earth year is only calculated once. You could consider introducing a constant for the `31557600.0` value.

Reply:
yes, true, this would be better